src/utils/get-core-member-logins.ts

```diff
@@ -26,8 +26,23 @@ export const getCoreMemberLogins = async (pull_number: number, teams?: string[])
   return uniq(teamsAndLogins.map(({ login }) => login));
 };
 
-export const getRequiredCodeOwnersEntries = async (pull_number: number): Promise<CodeOwnersEntry[]> => {
-  const codeOwners = (await loadOwners(process.cwd())) ?? [];
+export const getRequiredCodeOwnersEntries = async (pull_number: number, codeowners_overrides?: string): Promise<CodeOwnersEntry[]> => {
+  let codeOwners: CodeOwnersEntry[] = (await loadOwners(process.cwd())) ?? [];
+  if (codeowners_overrides) {
+    const codeOwnerOverrides: CodeOwnersEntry[] = codeowners_overrides.split(',').map(overrideString => {
+      const [pattern, ...owners] = overrideString.split(/\s+/);
+      if (!pattern) {
+        core.setFailed(
+          'Invalid code_owners_override format. Please provide a comma-separated list of lines in GitHub CODEOWNERS format. For example, "/foo @owner1 @owner2,/bar @owner3".'
+        );
+        throw new Error();
+      }
+      return { pattern, owners };
+    });
+    // codeowners-utils sorts its CodeOwnersEntry array in reverse order of the CODEOWNERS file
+    codeOwnerOverrides.reverse();
```

Comment:
```suggestion
    }).toReversed(); // codeowners-utils sorts its CodeOwnersEntry array in reverse order of the CODEOWNERS file
```

Reply:
@danadajian originally, i was going to use `.toReversed()`, but i was worried about Node compatibility -- looks like it [didn't land until Node 20](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed#browser_compatibility) which may exclude some consumers. happy to chain the `.reverse()` call if you'd prefer though.

Reply:
This action [runs on node 20](https://github.com/ExpediaGroup/github-helpers/blob/main/action.yml#L164), so we should be good

Reply:
FWIW, Jest still seems to use Node to run (despite being launched with bun), so if the user's default Node environment is <20, the tests will fail. However, moving forward since this repo's GitHub workflows seem to work fine.

Reply:
Node 18's end of life is [next month](https://github.com/nodejs/Release?tab=readme-ov-file#release-schedule) so honestly I'm not too worried about people using Node 18 😄